extensions/amp-story/1.0/test/test-page-advancement.js

```diff
@@ -129,6 +129,35 @@ describes.realWin('page-advancement', {amp: true}, (env) => {
 
         expect(advancement).to.not.be.instanceOf(TimeBasedAdvancement);
       });
+
+      it('should update delayMs_ when updateTimeDelay() is called', () => {
+        const pageEl = html`
+          <amp-story-page auto-advance-after="3s"> </amp-story-page>
+        `;
+        const advancement = AdvancementConfig.forElement(win, pageEl);
+        expect(advancement.delayMs_).to.be.equal(3000);
+
+        advancement.updateTimeDelay('5s');
+
+        expect(advancement.delayMs_).to.be.equal(5000);
+        expect(advancement.remainingDelayMs_).to.be.equal(null);
+      });
+
+      it('should update remainingDelayMs_ when updateTimeDelay() is called', () => {
+        const pageEl = html`
+          <amp-story-page auto-advance-after="3s"> </amp-story-page>
+        `;
+        const advancement = AdvancementConfig.forElement(win, pageEl);
+        // Simulate a pause after 1 second.
+        advancement.start();
+        advancement.startTimeMs_ -= 1000;
+        advancement.stop(true);
+        expect(advancement.remainingDelayMs_).to.be.equal(2000);
```

Comment:
@gmajoulet @Enriqe This test is flaky on `main`. Can someone take a look?

Example failures:
- https://app.circleci.com/pipelines/github/ampproject/amphtml/8866/workflows/2ea8df92-b56c-434a-98e3-f03f5b8d556f/jobs/134483
- https://app.circleci.com/pipelines/github/ampproject/amphtml/8873/workflows/8940beff-c45a-4835-9188-5997da23b5da/jobs/134598

/cc @ampproject/build-on-duty